roles/openldap/templates/password_policy_util.sh

```diff
@@ -0,0 +1,176 @@
+#!/bin/bash
```

Comment:
Is this file intended to be used outside of Ansible? If not, we shouldn't install it on the host, and instead turn it into an Ansible module we can use to accomplish the tasks.